worker/cloudflare-worker/src/worker.ts

```
@@ -0,0 +1,495 @@
+ * Determine which users should receive notifications for this event
+ */
+async function determineNotificationTargets(event: RelayEvent, env: Env): Promise<NotificationTarget[]> {
+  const targets: NotificationTarget[] = [];
+  
+  // Get currently online users (for normal priority events)
+  const onlineUsers = await getCurrentlyOnlineUsers(env);
+  
+  switch (event.kind) {
+    case 1: // Text note
+      // Check for mentions in tags
+      const mentions = event.tags.filter(tag => tag[0] === 'p').map(tag => tag[1]);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)